Command/CreateSiteCommand.php

```diff
@@ -48,15 +48,19 @@ public function configure()
 The <info>sonata:page:create-site</info> command create a new site entity.
 
 EOT
-);
+        );
     }
 
     /**
      * {@inheritdoc}
      */
     public function execute(InputInterface $input, OutputInterface $output)
     {
-        $dialog = $this->getHelperSet()->get('dialog');
+        if (!$this->getHelperSet()->has('question')) {
+            $helper = $this->getHelperSet()->get('dialog');
+        } else {
+            $helper = $this->getHelperSet()->get('question');
+        }
```

Comment:
Please provide `NEXT_MAJOR` instructions indicating what to do when dropping support for old versions of sf